### DISCUSSION TITLE
Umi项目打包后ModalForm组件的样式丢失

### DISCUSSION THREAD

**Discussion by Endergarten** [umijs/umi#10887] [Q&A]
在本地使用`umi dev`启动项目时，ModalForm组件渲染正常。将项目打包后部署，ModalForm组件渲染异常。检查打包后的css文件，发现其缺少`ant-modal-xxx`相关样式

**Comment by fz6m** [comment]
需要提供一个最小复现

**Reply by Endergarten** [reply]
您好，这是我使用umi dev启动项目后的效果:
![MRU~ZC X8I~GL4G{AFPRP~3](https://user-images.githubusercontent.com/69034333/229070687-a236ff66-f2e9-40e4-8c6e-87ae52c7904b.png)
这是我将项目打包后，使用本地服务器启动项目后的效果
![image](https://user-images.githubusercontent.com/69034333/229071735-779a9563-c2c7-4eb2-a394-8f714dead5e4.png)
以下是该页面的部分代码：
`import {`
`  ModalForm,`
`  ProFormGroup,`
`  ProFormText,`
`} from '@ant-design/pro-components';`
`import { Button, Form } from 'antd';
`
`export defauly (props: any) => {`
`  const [form] = Form.useForm();`
`  return (`
`    <ModalForm`
`      title="港口"`
`      trigger={<Button>1111</Button>}`
`      form={form}`
`      autoFocusFirstInput`
`      modalProps={{`
`        destroyOnClose: false,`
`        maskClosable: false,`
`      }}`
`      submitTimeout={1}`
`      labelCol={{ span: 6, offset: 2 }}`
`      layout="horizontal"`
`    >`
`      <ProFormGroup title="基本信息" labelLayout="twoLine">`
`        <ProFormText`
`          name="name"`
`          label="港口名称"`
`          wrapperCol={{ span: 8 }}`
`          width={'sm'}`
`        />`
`      </ProFormGroup>`
`    </ModalForm>`
`  );`
`};`
以下是`config.ts`部分配置文件：
`export default defineConfig({`
`  extraBabelPlugins: [`
`    [`
`      'import',`
`      {`
`        libraryName: 'antd',`
`        style: true, // 引入antd组件库的样式文件`
`      },`
`    ],`
`  ],`
`  chainWebpack: (config: any) => {`
`    config.optimization.minimize(false);`
`  },`
`  nodeModulesTransform: {`
`    type: 'none',`
`  },`
`  dva: {`
`    immer: true,`
`    hmr: false,`
`  },`
`  hash: false,`
`  fastRefresh: {},`
`});`

**Reply by ldwonday** [reply]
同样的问题，开发没问题，但是打包后没有，坑啊

**Comment by fz6m** [comment]
我用 umi 4 是没有这个问题的。

或者把 `extraBabelPlugins` 去了试试，但 umi 3 不维护了，所以建议升级到 umi 4。
